records_mover/url/base.py

```diff
@@ -209,16 +238,19 @@ def wait_to_exist(self,
         return
 
     def exists(self) -> bool:
+        # TODO: document
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker